lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -17,13 +17,25 @@
   }
 
   &:focus-within {
+    .nx-radio__circle {
+      filter: drop-shadow(0 0 3px rgba(2, 166, 217, 0.4));
```

Comment:
If this drop-shadow is going to be used across the board it should probably be moved to a variable or something.  Maybe the entire `filter` property should be moved to a mixin.

Reply:
> If this drop-shadow is going to be used across the board it should probably be moved to a variable or something. Maybe the entire `filter` property should be moved to a mixin.

I think that's a good idea, ironically the best place for it is `input-helpers.scss` where it is referenced in the very next mixin: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/69446e5e5883d8d68d0b1ac6f603cc94c2a9b966